lock.acquire() must be in the finally close

pypingou opened the pull request:
Otherwise the finally part is not being run I believe

pypingou committed:
lock.acquire() must be in the finally close

Otherwise the finally part is not being run I believe

pombredanne commented on docs/source/index.rst:
```diff
@@ -96,8 +96,8 @@ Examples
-    lock.acquire()
     try:
+        lock.acquire()
         pass
```
Would this `pass` still be needed then?

pypingou replied:
Good point, it is likely not :)

pypingou committed:
Drop un-needed ``pass`` instruction

Since there actually is something in the try/except, the ``pass`` that was there is no longer needed.
